src/__tests__/events.js

```diff
@@ -137,6 +137,12 @@ const eventTypes = [
   },
 ]
 
+const allEvents = eventTypes
+  .reduce((acc, curr) => {
+    acc.push(...curr.events)
+    return acc
+  }, [])
```

Comment:
This is great. Thank you for doing this.

I think we could simplify it:

```suggestion
const allEvents = eventTypes.flatMap(({events}) => events)
```

Reply:
While we're at it, I think we could/should export the `eventMap` itself and then import it into this file. I don't think we want to expose it from the library though so we'll need to be thoughtful about how we do that. Perhaps we could put the `eventMap` in a separate file and import it into both this file and the `events.js` file.

What do you think? I know it's more than you signed up for so if you'd rather not do that in this PR that's fine. We can do that later.

Reply:
Thanks for the feedback (and the great libraries)!

I avoided `flat`/`flatMap` in case there were concerns about [support](https://node.green/#ES2019-features-Array-prototype--flat--flatMap-) as the current node engines field is `>= 10`, but if not, I also prefer simpler.

> Perhaps we could put the eventMap in a separate file and import it into both this file and the events.js file

Had similar thoughts so, definitely agree.

The changes are straightforward, so it's no problem to do now, I think. :+1: